istioctl/cmd/dashboard.go

```diff
@@ -409,11 +412,16 @@ func closePortForwarderOnInterrupt(fw kube.PortForwarder) {
 		fw.Close()
 	}()
 }
-func openBrowser(url string, writer io.Writer) {
+func openBrowser(url string, writer io.Writer, skipOpen bool) {
 	var err error
 
 	fmt.Fprintf(writer, "%s\n", url)
 
+	if skipOpen {
+		fmt.Println("skip opening browser due to it is disabled by the --skipOpen option")
```

Comment:
Prefer to write to standard `io.Writer`.

```suggestion
		fmt.Fprint(writer, "skipping opening a browser")
```